### ISSUE TITLE
Add support for Web Workers

### ISSUE THREAD

**Issue by shoonia** [mckamey/countdown.js#37]
We can't use this library into Web Worker because we don't have any `module.exports` nor `window`.

We have en error: 
```console
ReferenceError: window is not defined
```

**Here:**

https://github.com/mckamey/countdownjs/blob/cb5b1877ca474d3c86e0407c090a7af90fbded41/countdown.js#L1355-L1364

## Suggestion of fix. 

Add check for window object

```diff
if (typeof module !== 'undefined' && module.exports) {
  module.exports = countdown;

- } else if (typeof window.define === 'function' && typeof window.define.amd !== 'undefined') { 
+ } else if (typeof window !== 'undefined' && typeof window.define === 'function' && typeof window.define.amd !== 'undefined') {
   window.define('countdown', [], function() {
    return countdown;
  });
}
```

**Comment by Valkyrihane** [comment]
I am working with `gatsby` on a static website. I had a similar problem after Gatsby upgraded to v3, where to window would not be defined during the html build, and if I added your check above, the `countdown()` function would be undefined in the `useEffect()`.

I ended taking the `countdown.js` file, removing the module and function scope, and exporting the `countdown` function at the end of the file like this: `module.exports = countdow;`, then importing it as a script in my project.

I am a bit sad to lose the dependency, but it works well, and was a quick fix.
